aws/resource_aws_cloudtrail_test.go

```diff
@@ -481,7 +482,7 @@ func testAccAWSCloudTrail_event_selector(t *testing.T) {
 		CheckDestroy: testAccCheckAWSCloudTrailDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccAWSCloudTrailConfig_eventSelector(cloudTrailRandInt),
+				Config: testAccAWSCloudTrailConfig_eventSelector(cloudTrailRandInt), // FLAG
```

Comment:
just curious about the 🏴 comment

Reply:
Oops, I left that there when I stashed some changes